setup.sh

```diff
@@ -209,6 +209,15 @@ LDAP_QUERY_FILTER_ALIAS=(&(kopanoAccount=1)(kopanoAliases=%s))
 LDAP_QUERY_FILTER_DOMAIN=(&(|(mail=*@%s)(kopanoAliases=*@%s)))
 SASLAUTHD_LDAP_FILTER=(&(kopanoAccount=1)(uid=%s))
 
+# LDAP user password self-service reset settings
+SELF_SERVICE_SECRETEKEY=$(random_string)
+SELF_SERVICE_PASSWORD_MIN_LENGTH=
```

Comment:
predefine some policies here?

Reply:
Which default policies would you like to see?

Reply:
Did miss this one before. Good question. The project itself defines it like this https://github.com/ltb-project/self-service-password/blob/master/conf/config.inc.php#L91-L105

I would say lets put all zeros there by default. having a number in there should at least make it easier for admins to see that this is something they could/should customise.

Reply:
The discussion of using proppery defaults is continued here:
https://github.com/zokradonh/kopano-docker/pull/70#discussion_r245516742